src/cryptography/hazmat/primitives/asymmetric/rsa.py

```diff
@@ -42,13 +42,30 @@ def public_key(self):
 
 
 @six.add_metaclass(abc.ABCMeta)
-class RSAPrivateKeyWithNumbers(RSAPrivateKey):
+class RSAPrivateKeyWithSerialization(RSAPrivateKey):
```

Comment:
Not really no. `RSAPrivateKeyWithNumbers` is already a subclass so I'd oppose changing that, but we could certainly choose to make its successor (`WithSerialization`) not a subclass and simply register both interfaces. The numbers classes are (I believe) the only place in cryptography that we utilize inheritance.

@alex, @public got opinions here? I'm happy to go either way.